iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ConfiguredBusImpl.java

```diff
@@ -69,6 +69,11 @@ public int getConnectedTerminalCount() {
         return getTerminals().stream().filter(Terminal::isConnected).map(Function.identity());
     }
 
+    @Override
+    public TerminalExt getTerminalReference() {
+        return getConnectedTerminalStream().findFirst().orElseGet(() -> getTerminals().isEmpty() ? null : getTerminals().get(0));
```

Comment:
I'm wondering, if there's no connected terminal, shouldn't we return null instead of taking a disconnected one?

Reply:
You're right. I change the algorithm to do what I wanted to do (find a terminal reference for a configured bus in bus-breaker linked to a bus with connected terminals by closed switches)